CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/MapViewTopComponent.java

```diff
@@ -197,15 +196,14 @@ protected void handleNewGraph(final Graph graph) {
                 }).get();
 
             } catch (final InterruptedException ex) {
-                Exceptions.printStackTrace(ex);
+                LOGGER.log(Level.SEVERE, "Exception thrown: " + ex.getMessage());
```

Comment:
can you change the log here to not use string contatentation? i.e.
`LOGGER.log(Level.SEVERE, "Exception thrown: {0}", ex.getMessage())`